src/Commands/RunScan.spec.ts

```diff
@@ -67,4 +86,128 @@ describe('RunScan', () => {
       expect(act).toThrowError(SyntaxError);
     });
   });
+
+  describe('handler', () => {
+    const runScan = new RunScan();
```

Comment:
Keep this one re-assigned. The reason is that before each test we have to ensure that we have clean state. We have `reset` that does it in `afterEach` for mocks but for sut itself we have to re-assign
```suggestion
    let runScan!: RunScan;
```